argus/action_manager/base.py

```diff
@@ -103,3 +96,26 @@ def wait_boot_completion(self):
     def specific_prepare(self):
         """Prepare some OS specific resources."""
         pass
+
+    @abc.abstractmethod
+    def exists(self, path):
+        """Check if the path exists."""
+        pass
+
+    @abc.abstractmethod
+    def is_file(self, path):
+        """Check if the file exits."""
+        pass
+
+    @abc.abstractmethod
+    def is_dir(self, path):
+        """Check of the directory exists."""
+        pass
+
+    def remove_file(self, path):
```

Comment:
It can be renamed as `remove` (like `os.remove` method).
Also why is not an abstract method ?